main.go

```diff
@@ -42,6 +42,7 @@ type tcpEventV4 struct {
 	Type  uint32
 	Pid   uint32
 	Comm  [16]byte
+	Ip    uint8
```

Comment:
probably is not necessary I did the porting from this https://github.com/kinvolk/bcc/blob/iaguis/tcp_tracer_established/tools/tcptracer.py#L45.

I will refactor it